coding-guides/making-your-own-starboard.md

```diff
@@ -191,7 +191,7 @@ if (stars) {
   const star = /^\⭐\s([0-9]{1,3})\s\|\s([0-9]{17,20})/.exec(stars.embeds[0].footer.text);
   const foundStar = stars.embeds[0];
   const image = message.attachments.size > 0 ? await this.extension(reaction, message.attachments.array()[0].url) : '';
-  const embed = new RichEmbed()
+  const embed = new MessageEmbed()
     .setColor(foundStar.color)
     .setDescription(foundStar.description)
     .setAuthor(message.author.tag, message.author.displayAvatarURL)
```

Comment:
> The avatar props have been moved to a function, yeah?

Reply:
They have, but this PR hasn't been updated yet, nor a new one opened like he said.